src/lib/accessibility/Behaviors/Popup/popupBehavior.ts

```diff
@@ -11,6 +11,7 @@ import * as _ from 'lodash'
  * Adds attribute 'aria-disabled=true' to 'trigger' component's part if 'disabled' property is true. Does not set the attribute otherwise.
  */
 const popupBehavior: Accessibility = (props: any) => {
```

Comment:
not related to this PR in particular, but could you, please, suggest if there is any reason for not introducing types for props in accessibility behaviors, i.e.:

```jsx
const popupBehavior: Accessibility = (props: PopupProps & PopupState) => {
```

This will alleviate prop rename refactoring changes, as well as clearly suggest what this input `props` arg should consist of.